docs/hyperparams.rst

```diff
@@ -115,10 +115,57 @@ Other options are:
 
 4. ``no_layers=2,5,6`` will generate three values - 2,5 and 6
 
-Note that option ``--hyperparam`` can be used several times for
+Note that option ``--hyperparam/-hp`` can be used several times for
 different hyperparameters; however, keep in mind that grid size grows
 exponentially with number of hyperparameters to try.
 
+Specifying hyperparameter ranges for plugin optimizers
+------------------------------------------------------
+Plugin optimizers are also supported. They can be enabled with the
+``-opt/--optimizer`` flag. Either the name of the optimizer (or its file path)
+can be specified as argument. The format is slightly different for plugin
+optimizers. Below are some examples:
+
+::
+
+    --hyperparam=lr=0:1:10:urla
+::
+
+    --hyperparam=lr=0:5:l
+::
+
+    --hp=lr=5:5:10:alu
+
+The general format is [min range]:[max range]:{array length}:{flags},
+where {array length} and {flags} are optional arguments. The following
+flags are supported:
+
+1. {u}: whether or not to constrain hyperparameters to [min range]:[max range]
```

Comment:
it would be handy to specify the default behaviour

Reply:
Done